templates/HAPROXY_BACKEND_HTTP_HEALTHCHECK_OPTIONS

```diff
@@ -0,0 +1,2 @@
+  option  httpchk GET {healthCheckPath}
```

Comment:
Good question. I guess the concern would be what happens when there is a disconnect between Marathon, the Python script, and HAProxy which results in leaving dead backends in the pool. For example, in the case of a network partition.